model/change_point_detection.go

```diff
@@ -464,8 +464,17 @@ func GetChangePointsGroupedByVersion(ctx context.Context, env cedar.Environment,
 		{
 			"$limit": args.PageSize,
 		},
+		// It is sad, that $group can't handle $skips and $limits :(
```

Comment:
take this out. 

aggregation works like a pipeline, so might be the fact that the skip/limit happens after $group? is there a way to project more of the document in the group to get out of this?

Reply:
projecting more of the documents inside the $group is what doesn't scale (since you have to group over the whole collection).

Reply:
IMO agg pipeline could have an optimization such that $group takes $skip and $limit downstream into account. But I digress, will remove the comment.